schema/deploy/database_functions/set_general_permissions.sql

```diff
@@ -0,0 +1,44 @@
+-- Deploy ggircs-portal:set_general_permissions to pg
+-- requires: schema_ggircs_portal
+
+begin;
```

Comment:
I'm worried that in maintenance we'll forget that this file is here and miss these permission declarations as we roll out access control. Let's push as much of this down to the deploy files they affect as possible